components/Cards/PictureCard/PictureCard.css

```diff
@@ -22,6 +22,7 @@
   height: 100%;
   background-color: var(--overlay-default);
   transition: all 200ms;
+  transform: translateZ(0);
```

Comment:
Nice find! Telling the browser to use the GPU ahead of time is a clever trick. Can we try `will-change: transform` instead? It has the same effect and I think it's more expressive.